modules/reference/pages/configuration-parameters.adoc

```diff
@@ -839,9 +839,13 @@ System.LogRoot) of log directory for Nginx |`+nginx+`
 
 |Nginx.ConfigTemplate |The template to generate nginx config. Please use
 `+@filepath+` to parse template from file. Check the default template
-first at https://docs.tigergraph.com.(Warning: Don’t modify the reserved
+first here. (Warning: Don’t modify the reserved
```

Comment:
This was auto generated from command-line help text. It's nice that you edited this, but the next time we generate it will still look like the original.